QA/README.md

```diff
@@ -0,0 +1,79 @@
+# CherryTwist - Quality Assurance
+
+This document provides overview of "Cherrytwist" quality assurance and testing activities. Some of them are in progress and some are to be started.
+
+# Process
+ 
+- Perform functional/non-functional testing types on reviewed "github" issues
+    - Depending on the "github" issue scope, testing is performed on different environments 
+- Log defects, communicate  and retest them
+- Perform regression testing as part of "github" issues testing
+- Automate/maintain test cases on different levels
+- Test data management
+
+
+# Testing types and levels
+
+"Cherrytwist" consists of multiple services. The following test types are being applied among the different test levels:
+
+## "Server" - testing types and levels:
+- ### Functional testing: 
+    - Unit testing<sup>*</sup>
```

Comment:
Do we not have any unit tests at the moment?

Reply:
The existing ones and autogenerated and I have to fix them. Once we have reasonable unit test suites for the different entities, I'll update the document.